notes/stories/television/tv-theamericanshortstory.st.txt

```diff
@@ -317,3 +317,47 @@ discrimination in society [The black farmhand Sulk was pointedly was forbidden f
+Productions for the Public Broadcasting Service. This installment is based on
+the 1898 Stephen Crane short story of the same name.
+
+Synopsis: A man encounters trouble during a stay at Nebraska hotel. 
+
+:: Date
+1977-04-12
+
+:: References
+https://www.imdb.com/title/tt0075771/
+
+:: Ratings
+4 <brent>
+4 <paul>
+
+:: Major Themes
+life in the American Wild West [The viewer is presented with a view of what it might have been like to stay at a small town hotel located on the American frontier circa 1898. In his monologue, Henry Fonda spelled out that the Swede got into trouble at the hotel on account of having let his fanciful, "dime-novel" inspired characterizations of the Wild West get the better of him.]
+overactive imagination [The Swede's fanciful, "dime-novel" inspired characterization of the Wild West quickly got him into trouble during his stay at the Fort Romper hotel.]
+xenophobia in society [The Swede had both fearful and derogatory misconceptions about the denizens of Fort Romper and those living in the Wild West in general, calling them "dangerous bastards" who would gang up on him and kill him. Bill and Johnnie derided the Swede for being an immigrant. Bill showed his contempt for foreigners by calling the Swede a Dutchman even after being corrected.] 
+self-fulfilling prophesy [Because of his overactive imagination, the Swede was convinced that he would be murdered in the hotel, and sure enough he ended up being killed by a fellow guest whom he'd provoked.]
+the hospitality industry [Mr. Scully was a small hotel owner and the story portrayed him at work. Mr. Scully did his utmost to be a paragon of hospitality so that his guests would be comfortable. He carried their bags from the station, helped them wash up, made sure they had a good dinner, and gave the Swede his prized gin from India to drink to calm his nerves. He also defended the Swede as a guest even though he displayed various types of boorishness.]
+hospitality [The hotel owner Mr. Scully did his utmost to be a paragon of hospitality so that his guests, the Swede in particular, would be comfortable during their stays.]
+
+:: Minor Themes
+cheating [Johnnie infuriated a local townsman and the Swede by cheating at the card game High Five.]
+tabletop games [The hotel guests played the card game High Five to pass the time.]
+father and son [Mr. Scully and his adult son Johnnie were running a frontier town hotel together.]
+journalism [One of the hotel guests said he was a journalist by trade.]
+fear for one's life [The Swede at one point clutched his cap in terror, thinking he was about to be killed.]
+inebriated interaction [The Swede got drunk and made a general ass of himself by verbally abusing everyone, leering at a photo of Mr. Scully's daughter, slamming down his cards, and awkwardly slapping Bill on the shoulder twice.]
+coping with the death of a child [Mr. Scully showed the Swede a photo of his dearly departed young daughter.]
```

Comment:
The Swede's dearly departed daughter?

Reply:
Disambiguated.

Reply:
Hmm I suspect it's Mr. Scully's daughter but it still looks like it might be the Swede's...